maat-court-data-api/src/main/java/gov/uk/courtdata/validator/MaatIdValidator.java

```diff
@@ -38,4 +41,21 @@ public Optional<Void> validate(final Integer maatId) {
         }
 
     }
+
+    /**
+     * @param maatId
+     * @return
+     * @throws ValidationException
+     */
+    public void validateNotExists(Integer maatId) {
+
+        if (maatId != null && maatId > 0) {
+            if (repOrderService.exists(maatId)) {
+                throw new ValidationException(String.format("There is already a record with this MAAT ID."));
```

Comment:
Should include the value of the `maatId` in the exception message

e.g.
`"There is already a record with MAAT ID [%d]".formatted(maatId)`